bin/ch-run.c

```diff
@@ -188,9 +187,10 @@ void env_delta_append(struct env_delta **ds, enum env_action act, char *arg)
    (*ds)[i].arg = arg;
 }
 
-/* Parse each line from --set-env at ':'. Each value is appended to the new
-   environment variable. '$' indicated that the variable will get expanded
-   except if --env-no-expand option is included. */
+/* Read input variable value pairs from --set-env. Split at '=' and tokenize
+   value on ':'. Each tokenize value append to new environment variable value.
+   '$' indicates that the variable will get expanded except if --env-no-expand
+   option is included. */
```

Comment:
```suggestion
/* Read input variable value pairs from --set-env and split on '='. Tokenize
   value on ':' and append each token to new environment variable value.
   Variables beginning with '$' are expanded unless the expand boolean is
   false. */
```

Reply:
Re. comment above, this would be my suggestion. However, I wouldn't get too hung up on this as @reidpr may (will) have even better suggestions.

Reply:
I would avoid verbs like "tokenize" because they're quite jargony. What does it mean to "tokenize on :"? I don't know.

That said, you may not need to say much about what this does since the behavior is pretty clear in the docs. It's legit to refer to those in comments.